cmd/cl001/ac001/execute/zz_generated.runner.go

```diff
@@ -45,7 +46,7 @@ func (r *runner) run(ctx context.Context, cmd *cobra.Command, args []string) err
 			Logger: r.logger,
 
 			KubeConfig:    env.KubeConfig(),
-			TenantCluster: env.TenantCluster(),
+			TenantCluster: config.Cluster("cl001", env.TenantCluster()),
```

Comment:
With this we consider the cluster ID in the process environment or the config file, in this order.